packages/nuxt-locokit/src/module.ts

```diff
@@ -182,7 +182,7 @@ export default defineNuxtModule<ModuleOptions>({
         name: ROUTES_NAMES.ADMIN.HOME,
         path: ROUTES_PATH.ADMIN.HOME,
         meta: {
-          protected: true,
+          // protected: true,
```

Comment:
What's the impact to comment this meta data?

If it's not used anymore, could you remove it for all routes ?

I think the goal of `protected` was in the router middleware / handler, to detect if the user is connected or not and if it needs an admin level ?

Reply:
My bad, I have forgotten to  put it back. I have done it in https://github.com/locokit/locokit/pull/228/files#diff-d069f548166c8cd08e9a8cf6d7a5a64616d0713e1dd6456483d6322475d84743